rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -91,9 +92,11 @@ bool RSocketStateMachine::resumeServer(
             << "Terminating new connection (" << frameTransport.get() << ")";
     frameTransport->closeWithError(folly::exception_wrapper(
         std::runtime_error("Old connection still exists")));
+    stats_->resumeFailure();
     return false;
   }
   CHECK(connect(std::move(frameTransport), resumeParams.protocolVersion));
+  stats_->resumeSuccess();
```

Comment:
I believe the bool value that is returned from the next line decides it the resumption is successful or not. So, we need to send resumeSuccess if the result of the next line is `true`.

Reply:
We might need one more `resumeFailure()` call at the inside of resumeFromPositionOrClose function.

Reply:
resumeFailure() or resumeSuccess() can be easily interpreted in RSocketServer.cpp.  Inside onRSocketResume() 
- if serverState->rSocketStateMachine_->resumeServer returns false or if result.hasError() then log resumeFailure()
- if serverState->rSocketStateMachine_->resumeServer returns true, then log resumeSuccess()

Similarly for the client side (RSocketClient.cpp), inside RSocketClient::resume()::ResumeCallback 
- onResumeOk() should log resumeSuccess()
- onResumeError() should log resumeFailure()

Reply:
Consider making the callbacks contain more information, I actually already did this in D5147228 (see stacked diffs for usage).